accounts/views.py

```diff
@@ -41,7 +43,13 @@ class SignUpView(generic.CreateView):
 class Home(LoginRequiredMixin, TemplateView):
     template_name = 'accounts/home.html'
 
+    def top_characters(self):
+        return Count.objects.filter(count_type=Count.CHARACTER).all().order_by('-count')[:5] 
+
+    def top_words(self):
+        return Count.objects.filter(count_type=Count.WORD).all().order_by('-count')[:5] 
+
     def dispatch(self, request, *args, **kwargs):
```

Comment:
In this case, isn't using @login_required equivalent to Django's LoginRequiredMixin [mixins.py](https://github.com/django/django/blob/301de774c21d055e9e5a7073e5bffdb52bc71079/django/contrib/auth/mixins.py) that I inherited from in the class?  Is there a general preference over one method or the other?  Thanks!